db/mainphotos copy.sql

```diff
@@ -0,0 +1,74 @@
+CREATE SCHEMA IF NOT EXISTS mainphotos;
```

Comment:
why is there a copy of the pqsl schema?

Reply:
That was a relic from trying different things with the schema creation, I'll get on deleting that, thanks!